source/server/admin/html/javascript.md

```diff
@@ -20,12 +20,13 @@ endpoints. This also disables active mode, which is dependent on HTML.
 
 ### Debugging
 
-To facilitate debugging and iterating on the JavaScript, a compile-time ifdef
-can be used at build time: `--cxxopt=-DENVOY_ADMIN_DEBUG`. When compiled this
-way, binaries (e.g. tests and envoy-static) will read the source files from
-their source tree locations every time they are served. So you can debug
-JavaScript by editing the file and refreshing the admin site in your browser,
-without rebuilding or restarting the binary.
+To facilitate debugging and iterating on the JavaScript, a simplified variant of
+envoy-static is provided in `test/integration/admin_html:test_server`, which
+accepts a `-debug` flag. When run with `debug` as its first argument,
```

Comment:
Which is it, a `-debug` flag, or `debug` as its first argument?
(The code says it's `-debug` as its first argument, so it looks like a flag but doesn't act like a flag. Maybe better to just make it `debug` as a first argument to avoid confusion, or to do proper flag handling.)

Reply:
good idea. done.